examples/draft-0-10-0/kentico/kentico-limited-height.html

```diff
@@ -0,0 +1,289 @@
+<!--
+Copyright (c) Facebook, Inc. and its affiliates. All rights reserved.
+
+This file provided by Facebook is for non-commercial testing and evaluation
```

Comment:
We might want to change the copyright... And information about by whom is the file provided...
Same for _kentico.html_.